### DISCUSSION TITLE
podrian agregar a los personajes de half life blue shift

### DISCUSSION THREAD

**Discussion by MaxQuiller** [VJ-HLR-Developers/Half-Life-Resurgence#48] [Ideas]
Hola, deberian añadir a los personajes de blue shift, para ser mas especifico, agreguen a barney sin la armadura (casco y chaleco) y al civ o civilian scientist, y que los 2 npc ya nombrados tengan la misma inteligencia que barney y el cientifico, solo que con diferente model :)

**Comment by javierlovessylvie** [comment]
amigo ellos hablan ingles no te van a entender pero si seria una buena idea esos npcs del blue shif
